queuelistener/listener.go

```diff
@@ -69,6 +71,9 @@ type Options struct {
 	// Log is used to log unexpected, non-fatal errors. It defaults to logging.DefaultLog.
 	Log logging.Logger
 
+	// Proxy protocol support
```

Comment:
As I stated in the comment on initialising proxy listener - I'd suggest adding `ProxyProtocolReadHeaderTimeout` or `ProxyProtocolTimeout` or something like this that would indicate what is this timeout. Don't forget to add appropriate comment so developers will know how does it work.

Reply:
I'm also not fully sure about naming. Should it be simple `ProxyProtocol` or `EnableProxyProtocol`. I saw in the `skipper.go` `Options` struct there are fields like `EnableRouteLIFOMetrics` or `EnableRatelimiters`. So it would follow the convention there.